Umbraco.gitignore

```diff
@@ -1,12 +1,21 @@
 # Note: VisualStudio gitignore rules may also be relevant
 
+# =========================
+# Umbraco
+# =========================
 # Ignore unimportant folders generated by Umbraco
 **/App_Data/ClientDependency/
 **/App_Data/ExamineIndexes/
 **/App_Data/Logs/
 **/App_Data/[Pp]review/
 **/App_Data/TEMP/
-Cached/
+**/App_Data/NuGetBackup/
+**/App_Data/FeedCache/
+**/App_Data/courier/revisions/
+**/App_Data/courier/cache/
+**/App_Data/umbraco.licensing.log.txt
+**/App_Data/*.udt
+**/aspnet_client/
```

Comment:
This can be removed. I've seen it generated from very old versions of Umbraco + Visual Studio. It's not part of the core and i haven't seen it for years so shouldn't be in this template.